src/modules/tas_studio/editor/mod.rs

```diff
@@ -285,6 +288,67 @@ struct AdjacentYawspeedAdjustment {
     bulk_count: usize,
 }
 
+/// Data for handling the side strafe accelerated yawspeed adjustment.
+#[derive(Debug, Clone, Copy)]
+pub struct MaxAccelYawOffsetAdjustment {
+    /// The mouse adjustment itself.
+    ///
+    /// (start, target, accel, yaw field value)
+    ///
+    /// Yaw field value here is the first value in the yaw field.
+    ///
+    /// It would be current yaw for yaw strafe or left-right count for left-right strafing.
+    mouse_adjustment: MouseAdjustment<MaxAccelYawOffsetMouseAdjustment>,
+    /// Indicates which change mode is in use.
+    ///
+    /// Left click while holding down right mouse button to switch change mode.
+    pub mode: MaxAccelYawOffsetMode,
+    /// Offset the mouse delta position when mode is switched.
+    mouse_offset: IVec2,
+    /// Skip Alt cycle if the mode is not available.
+    cycle_again: bool,
+}
+
+/// Modes of operation for maximum acceleration yaw offset adjustment.
+///
+/// Cycling by holding right mouse button and left click.
+#[derive(Debug, Clone, Copy)]
+pub enum MaxAccelYawOffsetMode {
+    StartAndTarget = 0,
+    Target,
+    Acceleration,
+    Start,
+    /// Alt is for adjusting the yaw field value of the strafe direction.
+    Alt,
```

Comment:
Should this maybe be renamed to YawField or StrafeDirYawField? And in the UI text, something like "Strafe Dir Yaw Field" too?